generators/entity/templates/server/src/main/java/package/domain/_Entity.java

```diff
@@ -88,11 +85,11 @@
 <%_         }
         }
 } if (databaseType === 'mongodb') { _%>
-@Document(collection = "<%= entityTableName %>")
+@org.springframework.data.mongodb.core.mapping.Document(collection = "<%= entityTableName %>")
```

Comment:
To make it less ugly when using mongo without ES I suggest to create an intance variable called mongoDocumentAnnotation and use it here the value of which can be qualified path when using ES and just document when using without ES. The same gies for ES annotation and the imports

Reply:
sure, I will take care of it